packages/daemon/src/daemon.js

```diff
@@ -676,6 +705,15 @@ const makeEndoBootstrap = (
             worker: provideValueForFormulaIdentifier(formula.worker),
           }),
         );
+      } else if (formula.type === 'lookup') {
+        return makeInspector(
+          formula.type,
+          formulaNumber,
+          harden({
+            agent: provideValueForFormulaIdentifier(formula.agent),
```

Comment:
I think this should be `store`.

Reply:
The property name, that is. `hub` for "name hub" would also be acceptable.